keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -914,8 +978,13 @@ For each of them, fill in the following information by copying the below templat
     - Testing: Are there any tests for failure mode? If not, describe why.
 -->
 
+No other failure modes are known.
```

Comment:
Maybe repeating the problem of rollback when ordinal is being used here would be good point, especially describing mitigations, diagnosting and testing bits?

Reply:
+1

Reply:
Done. Added a TODO for diagnostics. The TODO is to add logging to the StatefulSet controller (log a message when the `.spec.ordinals.start` field is set, but the feature is not enabled)